packaging/uenv.spec

```
@@ -33,5 +33,6 @@ echo "$REQ" > "$CNF"
 %license LICENSE
 %{_libdir}/libslurm-uenv-mount.so
 %{_bindir}/uenv
+%{_bindir}/squashfs-mount
```

```suggestion
%{_bindir}/squashfs-mount
%attr(4755, root, root) %{_bindir}/squashfs-mount
```

We might need to take additional steps to install the `squashfs-mount` binary as setuid.

The RPM script for the standalone `squasfs-mount` used the `install-setuid` option,
https://github.com/eth-cscs/squashfs-mount/blob/master/squashfs-mount.spec.in#L23
which called `chmod`/`chown` inside the makefile:
https://github.com/eth-cscs/squashfs-mount/blob/master/Makefile#L35

Apparently we shouldn't do this in the build/install step, and should instead specify the permissions that the file should be installed with here.